### PR TITLE
Improving root level readme

### PR DESCRIPTION
**Pull request by duckdegen**
Improving readme

### CODE REVIEW

**Comment by duckdegen** [comment]
@rhlsthrm plz provide some feedback. This will be updated more in the near future, but this is just to get a basic update out asap

File: README.md [hunk 1]
```diff
@@ -1,61 +1,134 @@
...
+<!-- ABOUT THE PROJECT -->
 
-**Nxtp** is a lightweight protocol for generalized xchain/xrollup transactions that retain the security properties of the underlying execution environment (i.e. it does **not** rely on any external validator set).
+## About Connext
 
-The protocol is made up of a simple contract that uses a locking pattern to `prepare` and `fulfill` transactions, a network of offchain routers that participate in pricing auctions and pass calldata between chains, and a user-side sdk that finds routes and prompts onchain transactions.
+Connext is a bridging protocol which allows for safe transfer of tokens and data between EVM compatible blockchains.
 
-## Transaction Lifecycle
+It's architecture can be seen as a layered system, as follows:
```
Comment (rhlsthrm) [inline README.md:70]:
```suggestion
The Connext architecture can be seen as a layered system, as follows:
```
"It's" is improper grammar.

File: README.md [hunk 2]
```diff
@@ -1,61 +1,134 @@
...
-   - Router can be _fully_ stateless.
-2. When the receiving side funds are unlocked, sender side is immediately able to be unlocked. This means it is not possible for liquidity to leak.
-3. All of our current functionality around crosschain transfers is preserved. Auctions and AMMs will work very easily on this.
-4. The protocol is stupidly simple (enough that we can reasonably build and test it in 2-3 weeks).
+-Emit an event with the tx details.
+Routers observing the origin chain with funds on the destination chain will:
+Simulate the transaction (if this fails, the assumption is that this is a more "expressive" crosschain message that requires permissioning and so must go through the slow Nomad process only).
 
-Drawbacks/Risks:
+-Prepare a signed transaction object using funds on the receiving chain.
```
Comment (rhlsthrm) [inline README.md:97]:
```suggestion
**Routers (Active Liquidity Providers) will:**
-Prepare a signed transaction object using funds on the receiving chain.
```
This is done by routers technically, not contracts.

File: README.md [hunk 3]
```diff
@@ -1,61 +1,134 @@
...
+Post this object (a "bid") to the auctioneer.
+Note: if the router does not have enough funds for the transfer, they may also provide only part of the transfer's value.
```
Comment (rhlsthrm) [inline README.md:99]:
```suggestion
Note: if the router does not have enough funds for the transfer, they may also provide only part of the transfer's value, which gets split across multiple routers in the network.
```

File: README.md [hunk 4]
```diff
@@ -1,61 +1,134 @@
...
 
-1. Nxtp is _only_ a protocol for (generalized) xchain transactions. It does not use channels (i.e. does not utilize offchain state). This means it cannot be used to do batched conditional transfers for the purposes of scalable micropayments.
-2. While there is great crash tolerance, there is a strong requirement that the router must reclaim its funds within a certain time window (we can set this how we like... presumably 48-96 hours). Note that the pessimistic channel case actually has this same liveness requirement, but it exists on both the user _and_ router.
+-The auctioneer will be observing all of the underlying chains. Every X blocks, the auctioneer will collect bids for transactions. The auctioneer will be responsible for selecting the correct router (or routers!) for a given tx (can be random). The auctioneer will post batches of these bids to a relayer network to submit them to chain.
```
Comment (rhlsthrm) [inline README.md:101]:
```suggestion
-The sequencer collects bids from routers and allows routers 30 seconds per transfer to send bids. After this time, sequencer will select a bid randomly (selection process TBD) and submit the payload which contains the router's signature to the relayer network to be submitted to chain.
```
Modified sequencer algorithm to be more accurate.

File: README.md [hunk 5]
```diff
@@ -1,61 +1,134 @@
...
+
+## About NXTP
+
+**Nxtp** is a liquidity layer and a developer interface on top of the [nomad](nomad-url) optimisitic briding protocol.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Structure of this repo
+
+This repo is set up as a yarn workspace monorepo
```
Comment (rhlsthrm) [inline README.md:127]:
```suggestion
This repo is set up as a Yarn Workspace monorepo: https://yarnpkg.com/features/workspaces
```

**Review by rhlsthrm — CHANGES_REQUESTED** [review]
Really great work here! This is awesome. Just posted a few suggestions to make things more accurate.

**Comment by duckdegen** [comment]
updated copy as per comments

**Comment by rhlsthrm** [comment]
Weird that builds are failing...

**Comment by rhlsthrm** [comment]
Might be bc of fork. Merged anyways.

**Pull request merged**